index.html

```diff
@@ -0,0 +1,137 @@
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link href="./style.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h3 class="logo-text">My Logo</h3>
+      <img
+        class="menu"
+        src="./images/menu.svg"
+        alt="triangle with all three sides equal"
+        height="87"
+        width="100"
+      />
+    </header>
+    <main>
+      <h1 class="header-text">I'm Abraham Rodriguez Martinez to see you!</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <img
+        src="./images/Ellipse-3.svg"
+        class="e-3"
+        alt="bird"
+        height="10"
+        width="51"
+      />
+
+      <img
+        src="./images/Ellipse-11.svg"
+        class="e-11"
+        alt="bird"
+        height="10"
+        width="51"
+      />
+      <img
+        src="./images/Rectangle-4.svg"
+        class="r-4"
+        alt="bird"
+        height="38"
+        width="20"
+      />
+      <img
+        src="./images/Ellipse-5.svg"
+        class="e-5"
+        alt="bird"
+        height="11"
+        width="10"
+      />
+      <img
+        src="./images/Rectangle-7.svg"
+        class="r-7"
+        alt="bird"
+        height="11"
+        width="10"
+      />
+      <img
+        src="./images/Ellipse-8.svg"
+        class="e-8"
+        alt="bird"
+        height="15"
+        width="15"
+      />
+      <img
+        src="./images/Rectangle-6.svg"
+        class="r-6"
+        alt="bird"
+        height="20"
+        width="96"
+      />
+      <img
+        src="./images/Ellipse-8.svg"
+        class="e-8-b"
+        alt="bird"
+        height="15"
+        width="15"
+      />
+
+      <section class="social">
+        <h6 class="social-text">LET'S CONNECT</h6>
+        <div class="social-icons">
+          <img
+            src="./images/Vector-2.svg"
+            class="icon"
+            alt="bird"
+            height="20"
+            width="20"
+          />
+          <img
+            src="./images/Linkedin.svg"
+            class="icon"
+            alt="bird"
+            height="20"
+            width="20"
+          />
+          <img
+            src="./images/m.svg"
+            class="icon"
+            alt="bird"
+            height="20"
+            width="20"
+          />
+          <img
+            src="./images/Vector-1.svg"
+            class="icon"
+            alt="bird"
+            height="20"
+            width="20"
+          />
+          <img
+            src="./images/Vector.svg"
+            class="icon"
+            alt="bird"
+            height="20"
+            width="20"
+          />
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)